front/src/reducers/editor/thunkActions.ts

```diff
@@ -79,32 +79,37 @@ export function updateTotalsIssue(infraID: number | undefined) {
       let filterTotal = 0;
       if (infraID) {
         // Get total
-        const totalResp = dispatch(
-          osrdEditoastApi.endpoints.getInfraByIdErrors.initiate({
-            id: infraID,
-            level: 'all',
-            errorType: undefined,
-            pageSize: 1,
-            page: 1,
-          })
+        const totalResp = await dispatch(
+          osrdEditoastApi.endpoints.getInfraByIdErrors.initiate(
+            {
+              id: infraID,
+              level: 'all',
+              errorType: undefined,
+              pageSize: 1,
+              page: 1,
+            },
+            { subscribe: false, forceRefetch: true }
+          )
         );
-        totalResp.unsubscribe();
-        const totalResult = await totalResp;
-        total = totalResult.data?.count || 0;
+        total = totalResp.data?.count || 0;
+        filterTotal = total;
 
-        // Get total for the active filters
-        const filterResp = dispatch(
-          osrdEditoastApi.endpoints.getInfraByIdErrors.initiate({
-            id: infraID,
-            level: editor.issues.filterLevel,
-            errorType: editor.issues.filterType ?? undefined,
-            pageSize: 1,
-            page: 1,
-          })
-        );
-        filterResp.unsubscribe();
-        const filterResult = await filterResp;
-        filterTotal = filterResult.data?.count || 0;
+        // Get total for the active filters (if needed)
+        if (editor.issues.filterLevel !== 'all' || editor.issues.filterType !== undefined) {
```

Comment:
It looks like when filter type is on "All", he `filterType`property is `null` and not `undefined`. So when entering the infra editor, instead of seeing  "X anomalies" at the bottom right, we see this : 
![image](https://github.com/OpenRailAssociation/osrd/assets/113104137/d83b2381-3c30-43da-8a80-dbe4d28631f4)

```suggestion
        if (editor.issues.filterLevel !== 'all' || editor.issues.filterType !== null) {
```

Reply:
I used `isNil` from lodash which handles both cases